section1/reflection.md

```diff
@@ -2,18 +2,48 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+*I struggle with asking questions due to bad experiences in college/graduate school. It's never nice to be shamed for asking "stupid" questions. Nowadays, I tend to wait until I'm completely desperate before asking anyone. I know this is something I will need to work on during my time at Turing.*
+
 2. What are the data types you learned about in this section? In your own words, define each.
 
-3. How would you log the string `"Hello World!"` to the console?
+ * *Strings - data in letters or characters (can be numbers too!) as long as enclosed in quotes.*
+ * *Boolean - data that is either 'true' or 'false'.*
+ * *Number - data that includes all numbers ( includes negatives/decimals)!*
+
+3. How would you log the string `"Hello World!"` to the console?;
+
+``` JavaScript
+var message = "Hello World!";
+console.log(message);
 
+```
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+* *(//) Backslashes for 'single line comments.*
+* *(/*) *Backslash + asterisk on both ends of comments if there are multiple lines.*
+
+*Both are used to indicate code comments, which explain what your code does. It helps your future self as well as other collaborators understand your intended code meaning/function, especially after the passing of time.*
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+*A variable is a storage container for a value with a specific data type: a number, a string (words or other characters), or a Boolean (true or false). It allows the computer to easily store and access information.*
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+*Weather.com app; temperature-number, humidity-number, and windSpeed-number.*
+
 7. In your own words, explain what concatenation is.
 
+*Concatenation is putting strings and values together with the (+) symbol to form messages, statements, or information to be logged into the console.*
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+*Amazon welcome page where they greet you by name.
+Example:
+```JavaScript
+var name = 'Lidia';
+console.log("Hello," + name);
+```
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+*Which is best practice: interpolation or concatenation? Does it matter?*
```

Comment:
Great questions! I wouldn't say one is better; concatenation is older, interp is newer, so usually people lean towards the new shiny things. I would say interpolation is more frequently used in modern JavaScript. At the end of the day, most important thing is to use one consistently!